gulpfile.js

```diff
@@ -16,6 +16,7 @@ var gulp   = require('gulp'),
 			'components/prism-css.js',
 			'components/prism-clike.js',
 			'components/prism-javascript.js',
+			'components/prism-solidity.js',
```

Comment:
Please remove this change. This array lists the components included by default in the `prism.js` file used of the website. Solidity does not need to be added there.